rero_ils/modules/ext.py

```diff
@@ -71,7 +72,7 @@ def __init__(self, app=None):
             app.add_template_filter(can_edit, name='can_edit')
             app.add_template_filter(
                 can_access_item,
-                name='can_access_item'
+                name='can_access_itetest_patron_organisation_pidm'
```

Comment:
it seems `test_patron_organisation` string has been placed into 'ite' and 'm'. Bad copy/paste ?